files/scripts/swss.sh

```diff
@@ -138,6 +142,9 @@ stop() {
     # if warm start enabled or peer lock exists, don't stop peer service docker
     if [[ x"$WARM_BOOT" != x"true" ]]; then
         /bin/systemctl stop ${PEER}
+        for dep in ${DEPENDENT}; do
+            /bin/systemctl stop ${dep}
```

Comment:
suggest to add function stopPeerDependentServices() and put both in that function.

Reply:
Thanks. Done.